core/src/main/java/tc/oc/pgm/command/MatchCommand.java

```diff
@@ -172,6 +167,13 @@ public void match(Audience viewer, CommandSender sender, Match match) {
     }
   }
 
+  private int getNonVanishedCount(Collection<MatchPlayer> players) {
+    return players.stream()
+        .map(MatchPlayer::getBukkit)
+        .mapToInt(p -> Integration.isVanished(p) ? 0 : 1)
+        .sum();
```

Comment:
```suggestion
        .filter(Integration::isVanished)
        .count();
```

Reply:
Wouldn't this achieve the opposite? Since filtering for just vanished would return the vanished count and not the non-vanished count. The purpose of the method is to calculate the number of players in a team who are _not_ vanished.

```java
  private int getNonVanishedCount(Collection<MatchPlayer> players) {
    return players.stream()
        .map(MatchPlayer::getBukkit)
        .mapToInt(p -> Integration.isVanished(p) ? 0 : 1)
        .sum();
  }
```

An alternative approach that aligns with your suggestion could be:
```java
private int getNonVanishedCount(Collection<MatchPlayer> players) {
      return players.size() - Math.toIntExact(players.stream()
                      .map(MatchPlayer::getBukkit)
                      .filter(Integration::isVanished)
                      .count());
  }
```
Though the `mapToInt` method was a little more clean. Let me know which version should be used and I'll make the change if needed. Thanks again for the feedback!

Reply:
I didn't realize it was backwards, then the proper solution would be:

```suggest
        .filter(p -> !Integration.isVanished(p))
        .count();
```